src/app/issues/PageIssues.tsx

```diff
@@ -126,11 +126,17 @@ export const PageIssues = () => {
   const { t } = useTranslation();
   const [search, setSearch] = useState('');
   const [filters, setFilters] = useState<{ scopes: string[] } | null>(null);
+  const [lastFilters, setLastFilters] = useState<string[] | null>([]);
   const [selectedIssues, setSelectedIssues] = useState<string[]>([]);
 
   const brandColor = useToken('colors', 'brand.500');
   const { isOpen, onClose, onOpen } = useDisclosure();
 
+  const onCloseExport = (scopes: string[] | undefined) => {
+    onClose();
+    setLastFilters(scopes ?? []);
```

Comment:
With the preview of the exported issues, this is not very useful, we can remove that.